conf/application.conf

```diff
@@ -326,7 +326,13 @@ db {
     index = "bi-dev"
     index = ${?ONS_BI_API_INDEX_NAME}
 
-    local = false // true if you want elastic in the same JVM - test mode
-    local = ${?ONS_BI_ES_LOCAL}
+    recreateIndex = false
+    recreateIndex = ${?ONS_BI_ES_RECREATE_INDEX}
+
+    loadTestData = false
+    loadTestData = ${?ONS_BI_ES_LOAD_TEST_DATA}
+
+    csvFilePath = "conf/demo/sample.csv"
+    csvFilePath = ${?ONS_BI_ES_CSV_FILE_PATH}
   }
 }
```

Comment:
An alternative I suppose would have been to have kept this out of the Play configuration - and to have provided a standalone main method that accepted the input file as an argument and created & populated the index accordingly.